ui/bootstrap/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@craftercms/bootstrap-adapter",
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "Jose V.",
+  "license": "LGPL-3.0-only",
+  "private": true,
+  "scripts": {
+    "build": "chmod +x build.sh; ./build.sh"
```

Comment:
This can probably be done via simple gradlew copy. Preferably we would avoid sh scripts.